source/slang/slang-ir-diff-jvp.cpp

```diff
@@ -144,6 +144,14 @@ struct DifferentiableTypeConformanceContext
     // 
     IRInst* getDifferentialForType(IRBuilder* builder, IRType* origType)
     {
+        switch (origType->getOp())
```

Comment:
Should this switch-case be here explicitly? I think the extensions defined in diff.meta.slang can handle these types?

Reply:
Yeah, I added this thinking that it might be good to have a fast path for builtin types, but this is not necessary.